### ISSUE TITLE
Add to Windows Store  

### ISSUE THREAD

**Issue by musm** [cormullion/juliamono#76]
This is more of a feature request (even if lofty) 

But it would be cool if this was in the Windows Store like a few other Windows fonts, which would allow auto-updates. https://docs.microsoft.com/en-us/windows/uwp/publish/

**Comment by cormullion** [comment]
Could be interesting... It's not a lot of money, but it's not quite the open-source way...

```Individual accounts cost approximately $19 USD```

**Comment by musm** [comment]
Yeah, I usually never remember to update my fonts. But the auto-update feature has been very, very convenient with Cascadia Code and now I never have to manually keep updating my fonts, which why I thought it would be a great enhancement here.

**Comment by cormullion** [comment]
I wonder if it could be a VS Code add-on. Have to investigate...

**Comment by musm** [comment]
What advantage does that bring?  In other words,  JuliaMono isn't a code editor extension so I don't see how this would help.

**Comment by cormullion** [comment]
Yes, true. I know there's a kind of extensions store, but perhaps it doesn't apply here...

**Comment by archelpeg** [comment]
I simply subscribed to new releases, so whenever there is a new version, I get an email and update it. Works for me.

**Comment by cormullion** [comment]
Thanks - and please let me know if there are any regressions, I can't test it on Windows... ! :)

**Comment by RoyiAvital** [comment]
@cormullion , Could you please specify which one?
Should Windows users use `JuliaMono.zip`, `JuliaMono-ttf.zip` or `JuliaMono-webfonts.zip` (I guess not)?

**Comment by cormullion** [comment]
Well if you want just the web fonts - to serve to a web site, for example - download the `webfonts.zip` or `webfonts.tar.gz`, depending on which format you prefer. If you want just the TTF fonts, then `JuliaMono-ttf.zip` or `.tar.gz` looks like the one to go for. For everything, go for `JuliaMono.zip` or `.tar.gz`.

Having said that, the size of the `JuliaMono-ttf.zip` and `JuliaMono.zip` files look suspiciously similar. Perhaps there's an improvement to be made in the github actions file https://github.com/cormullion/juliamono/blob/master/.github/workflows/ci.yml . If you find a way to improve it, go for it!

**Comment by RoyiAvital** [comment]
So there are 2 flavors, Web and TTF? So for the context of the OS only the TTF are needed?

**Comment by cormullion** [comment]
> So there are 2 flavors, Web and TTF?

yes, if you don't know what web fonts are, you don't need them 😀

**Comment by RoyiAvital** [comment]
OK, so this is what caused my confusion, I thought there are 3: TTF, WebFonts and the 3rd zip. I didn't know, excluding the web fonts what suits the OS. So maybe the documentation should say that for OS level one only needs the TTF variant and the ZIP without a variant basically has both.

Thanks for this great work/
